pages.id/common/ruby.md

```diff
@@ -0,0 +1,24 @@
+# ruby
+
+> Interpreter bahasa pemrograman Ruby.
+> Informasi lebih lanjut: <https://www.ruby-lang.org>.
+
+- Memulai REPL (_shell_ interaktif):
```

Comment:
I'm not sure if we allow `_something_` syntax in tldr.

Reply:
Yeah, we accept that. We don't have an strict rule and I understand the author uses this for English words.

Reply:
If doesn't throw any error, do you think it's still okay to leave it like that until `tldr` supports the syntax? I mean it's still a valid markdown. wdyt?